data/fleets.txt

```diff
@@ -2784,38 +1864,25 @@ fleet "Marauder fleet IX"
 		plunders
 	variant 3
 		"Marauder Falcon"
-		"Osprey"
-		"Osprey (Missile)"
-		"Osprey (Laser)"
+		variant "Osprey" 3
 	variant 1
 		"Marauder Falcon (Weapons)"
-		"Osprey"
-		"Osprey (Missile)"
-		"Osprey (Laser)"
+		variant "Osprey" 3
 	variant 1
 		"Marauder Falcon (Engines)"
-		"Osprey"
-		"Osprey (Missile)"
-		"Osprey (Laser)"
+		variant "Osprey" 3
 	variant 3
 		"Marauder Leviathan"
-		"Firebird"
-		"Firebird (Plasma)"
-		"Firebird (Missile)"
+		variant "Firebird" 3
 	variant 1
 		"Marauder Leviathan (Engines)"
-		"Firebird"
-		"Firebird (Plasma)"
-		"Firebird (Missile)"
+		variant "Firebird" 3
 	variant 1
 		"Marauder Leviathan (Weapons)"
-		"Firebird"
-		"Firebird (Plasma)"
-		"Firebird (Missile)"
+		variant "Firebird" 3
 	variant 1
 		"Marauder Leviathan (Weapons)"
-		"Firebird (Laser)" 3
-		"Firebird (Missile)"
+		variant "Firebird" 4
```

Comment:
The reviewers for data-files and code-changes are different;
This remark is a bit too late for this PR (since you already changed a very large amount of datafiles), but for future PRs please consider to keep the code-change together with a small example data-file change in one PR and to file the data-file changes in a separate PR.
(You can mark this remark with "Conversation Handled" after you read this comment.)

Reply:
Yeah, I was actually considering earlier removing the data changes and adding those separately after the code changes are merged, which I'll likely do so that the comments I made on the data changes can be discussed more, and since the code changes alone retain the current functionality, meaning no data change is actually necessary.